posts/2023-06-29-OutreachyInternshipExperience/index.qmd

```diff
@@ -0,0 +1,235 @@
+nature of the internship: we had to contribute to projects of our own choice for
+a month, and that meant that whether or not I got the internship, I would
+definitely get a chance to learn and interact with the open-source community!
+Once my initial application period was over, I took a few days to go over the
+many projects listed on the platform, for which we could choose to contribute.
+
+### The Microbiome Study Curation Project at Bioconductor
+
+A lot of the projects were very appealing but what stood out to me was the
+‘Microbiome Study curation’ project at [BugSigDb](bugsigdb.org) ; for me, it was
+quite different to the others and while it wasn’t extremely tedious, it was
+fascinating to learn about the project: I realized how I wasn’t even properly
+aware of how epidemiological studies and research on samples of varying sizes
+and nature are critical for medical research, how they play a crucial role in
+medical research by providing insights into the distribution, causes, and
+prevention of diseases within populations. These studies involve the collection,
+analysis, and interpretation of data from large populations over time, and can
+help to identify risk factors, patterns of disease occurrence, and potential
+interventions that can improve public health.
+
+The [Outreachy contribution
+period](https://www.outreachy.org/docs/applicant/#make-contributions) was one of
+the best experiences I have ever had as a student. I genuinely loved making
+contributions and learning. My mentors, Chloe and Fatima as well as the BugSigdb
+community- Professor Levi and others always helped me out whenever I got stuck
+on a problem or had an issue in understanding something.
+
+## About the Project
+
+The Microbiome Study curation project mainly deals with reviewing existing
+epidemiological studies on BugSigdb and curating new studies on BugSigdb.
+
+Throughout my internship, the contributions I have made include:
+
+- Reviewing existing curations of epidemiological studies on Bugsigdb
+- Finding and curating more epidemiological studies
+- Learning about the data analysis process using R and implementing it
+
+It has been a great source of knowledge for me, and has broadened my views as
+well as technical skills for understanding scientific articles, conducting a
+literature review and carrying out data analysis. The wonderful community at
+Bioconductor has only always helped me out and has provided me with a richer
+understanding of everything I've been doing! I'm really glad to be a part of
+this project. If you want to know more, I would love to introduce to you my
+[blog](https://atrayees0.wordpress.com/) , where I record my progress as well as
+my experiences and learnings.
+
+## Upcoming Opportunities at Outreachy and Bioconductor!
+
+Applicants who are interested in this amazing Outreachy internship for the
+December 2023 cohort can [apply](https://www.outreachy.org/apply/) of the
```

Comment:
Replace 'of' with 'for'

    December 2023 cohort can [apply](https://www.outreachy.org/apply/) for the